sensor_msgs/Image depth_image
sensor_msgs/Image rgb_image
---
bool success
template_algo/Grasp2D best_grasp
